src/Utilities/copy/allocate_load.jl

```diff
@@ -0,0 +1,445 @@
+# TODO deprecate this in MOI v0.7
+"""
+    needs_allocate_load(model::MOI.ModelLike)::Bool
+
+Return a `Bool` indicating whether `model` does not support `add_variables`/
+`add_constraint`/`set` but supports `allocate_variables`/`allocate_constraint`
+/`allocate`/`load_variables`/`load_constraint`/`load`.
+That is, the allocate-load interface need to be used to copy an model to `model`.
+"""
+needs_allocate_load(::MOI.ModelLike) = false
```

Comment:
I can't find this used anywhere. Is it safe to remove @blegat?

Reply:
Yes, there is a todo to remove it